core/.changelog.d/2475.fixed

```diff
@@ -0,0 +1 @@
+Do not show "USB NOT CONNECTED" if device enters USB suspend state from configured state
```

Comment:
please remove this (or reword the previous changelog entry, if needed)

this way it would needlessly spam the changelog for a feature that has not changed between releases

Reply:
I'll add [no changelog] instead.